lua/telescope/actions/init.lua

```diff
@@ -299,6 +299,23 @@ actions.select_tab = {
   end,
 }
 
+--- Perform 'tab drop' action on selection, usually something like<br>
+---`:tab drop <selection>`
+---
+--- i.e. open the selection in a new tab
+---@param prompt_bufnr number: The prompt bufnr
+actions.select_tab_drop = {
+  pre = function(prompt_bufnr)
+    action_state.get_current_history():append(
+      action_state.get_current_line(),
+      action_state.get_current_picker(prompt_bufnr)
+    )
+  end,
+  action = function(prompt_bufnr)
+    return action_set.select(prompt_bufnr, "tab drop")
```

Comment:
not sure if i like this key with a ` `,  maybe `_` is a little bit better

Reply:
https://github.com/rkoval/telescope.nvim/commit/87557efcd1a1475dcb436302fbeac5bd9150b14c